packages/backend/migrations/20240916181643_require_estimated_exchange_rate.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
+  return knex('quotes')
+    .whereNull('estimatedExchangeRate')
+    .update({
+      // TODO: vet this more... looks like the low* fields were (inadvertently?)
+      // made nullable when updating from bigint to decimal. If they are null
+      // anywhere then this wont work.
+      estimatedExchangeRate: knex.raw('?? / ??', [
+        'lowEstimatedExchangeRateNumerator',
+        'lowEstimatedExchangeRateDenominator'
+      ])
+    })
+    .then(() => {
+      return knex.schema.table('quotes', (table) => {
+        table.decimal('estimatedExchangeRate', 20, 10).notNullable().alter()
+      })
+    })
+}
```

Comment:
Could these actually be null in prod? They were originally not null and then converted from bigint to decimal where the not null constraint was (I think unintentionally?) lost

Reply:
see the TODO comment

@mkurapov can you comment on if dropping the not nullable constraint on `lowEstimatedExchangeRateNumerator` and `lowEstimatedExchangeRateDenominator` were important when the type was changed? im guessing it was inadvertent and we _should_ always be setting it

@raducristianpopa would you be able to verify if any of `lowEstimatedExchangeRateNumerator` or `lowEstimatedExchangeRateDenominator` values are null in testnet?